### ISSUE TITLE
Specify version of LibreOffice

### ISSUE THREAD

**Issue by davidbasalla** [gotenberg/gotenberg#452]
Is there currently a way to specify the version of LibreOffice to use? I don't think so from a brief look at the code - it looks like the latest libreoffice is being pulled in the Dockerfile. 

Motivation: It looks like the latest version of LibreOffice (v7.3) has some regressions and I would like to use a previous version (v7.1). v7.3 turns a 20 page DOCX into a 40 page DOCX (the spacing is doubled somehow between paragraphs), so the resulting PDF from Gotenberg is also 40 pages long. This is not the case in 7.1.

**Comment by gulien** [comment]
Helllo @davidbasalla,

I do agree there are regressions with LibreOffice v7.3. Check the following comment for more details: https://github.com/gotenberg/gotenberg/issues/442#issuecomment-1082028615

TL;DR: there are currently now easy way to retrieve version `v7.1`, AFAIK.
